AmericanMedicalAssociation/ama-style-guide-2
# ADF-131:  Loading gif

User story [ADF-131: Prototype search results page footer / load more](https://palantir.atlassian.net/browse/ADF-131)

## Desc
Add loading gif to styleguide.

## Testing
* Visit `http://localhost:3000/?p=atoms-loading` and verify a loading gif exists.
* Visit `http://localhost:3000/?p=pages-dr-finder-with-search-results` for display.
